microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -687,17 +687,17 @@ To enable new functions for randomly selected users, Microsoft Edge regularly se
 
 More controls and configurations are available for organizations.  For more information about controls and configurations for organizations, see [Microsoft Edge configurations and experimentation](/deployedge/edge-configuration-and-experiments).
 
-As a user, you aren't able to turn off the browser updates controlled or configured by your organization. You can control whether your product usage data is sent to Microsoft. Go to `edge://settings/privacy`, and change the **Optional diagnostic data** settings.
+As a user, you aren't able to turn off the browser updates controlled or configured by your organization. You can control whether your product usage data is sent to Microsoft. Go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> and change the **Optional diagnostic data** settings.
```

Comment:
correct.

Reply:
You can control whether your product usage data is sent to Microsoft: select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then change the **Optional diagnostic data** settings.